custom_model_runner/datarobot_drum/drum/drum.py

```diff
@@ -942,6 +943,24 @@ def _prepare_docker_command(self, options, run_mode, raw_arguments):
             )
             docker_cmd_args += " -p {port}:{port}".format(port=port)
 
+        if (
```

Comment:
from our discussion use any of:

```python
        if getattr(options, ArgumentsOptions.RUNTIME_PARAMS_FILE, None):
            docker_cmd_args += f' -v "{options.runtime_params_file}":{in_docker_runtime_parameters_file}'
            DrumUtils.delete_cmd_argument(in_docker_cmd_list, ArgumentsOptions.RUNTIME_PARAMS_FILE)
            in_docker_cmd_list.extend([ArgumentsOptions.RUNTIME_PARAMS_FILE, in_docker_runtime_parameters_file])
```

Reply:
`ArgumentsOptions.RUNTIME_PARAMS_FILE` -> "--runtime-params-file", while options contains "runtime_params_file".

I updated this to use 
```
if getattr(options, ArgumentOptionsEnvVars.RUNTIME_PARAMS_FILE.lower(), None):
```